token-client/src/main/java/com/sap/cloud/security/xsuaa/client/OAuth2TokenService.java

```diff
@@ -5,6 +5,9 @@
  */
 package com.sap.cloud.security.xsuaa.client;
 
+import com.sap.cloud.security.client.ClientCredentials;
```

Comment:
for better understanding, i would replace all ClientCredentials occurences in arguments and javadoc to ClientIdentity to avoid misunderstanding and as it is compatible

Reply:
change type to ClientIdentity